ansible/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+# tasks file for ocemr
```

Comment:
Any further thoughts about a shared infrastructure? What about if we use githubs releases feature, to tag and upload debs to github? It wouldn't be apt repos tho... so we'd have to verify packages another way? Or I suppose, we could just setup an s3 (http site) with a repo, and allow multiple people signing rights?

Reply:
Yup. a shared S3 site would probably be best (and cheapest). How should we set something up?